maven/bnd-export-maven-plugin/pom.xml

```diff
@@ -41,6 +41,10 @@
             <groupId>biz.aQute.bnd</groupId>
             <artifactId>biz.aQute.repository</artifactId>
         </dependency>
+        <dependency>
+            <groupId>biz.aQute.bnd</groupId>
+            <artifactId>bnd-maven-plugin-lib</artifactId>
```

Comment:
The parent is just a *reactor*. I think it would make it painful to put it in the parent.